### DISCUSSION TITLE
Eth Stuck - Failed to run simulation: AccountNotFound - PLEASE HELP!!!

### DISCUSSION THREAD

**Discussion by KrisWilka** [wormhole-foundation/wormhole#541] [Developer Q&A] [answered]
I have been trying to transfer AeWeth 0.4 (Allbridge from Ethereum) from Solana (Phantom) to ethereum (metamask) via the wormhole bridge, however the transaction failed due to the fee payer account not existing somehow I'm not sure how this works. I try to run the simulation again and get the same message: Account not found. Now my Crypto is stuck and I do not know how to retrieve it. I can imagine it being simple or running the transactions again but replacing this non-exsisiting account but unsure how to do so. If some developer could please help me out in this situation it would be massively appreciated. 

Below is what is states after I try and run the simulation again on Wormhole Bridge transaction inspector.

Fee payerSignerWritable | Cx56USuT9rYGqmJ8aU8ipNkhvs3jA8kFXWS3HPmJUj71. Account doesn't exist
-- | --

Here are the details: 

Signature: 5p8cT6j53zmHL5Scwx4rfQ9x9hqPWBS987JbVMgBi6WtusU7WTWo6QQXgq72mL62WkLKzLCFvBQ5WnsK9xE4LT6y

Solana Source Wallet: Cx56USuT9rYGqmJ8aU8ipNkhvs3jA8kFXWS3HPmJUj71
Eth Destination Wallet: 0x3F3fDC77A02B5EeEB07b792B8394A89f65bE1476

<img width="1180" alt="Screenshot 2021-10-14 at 23 47 43" src="https://user-images.githubusercontent.com/30329690/137405917-86525978-aeef-45ee-bd29-24fd2d29c4c9.png">
<img width="1210" alt="Screenshot 2021-10-14 at 23 48 19" src="https://user-images.githubusercontent.com/30329690/137405924-3802df86-d431-4728-86f6-3801c079e1f0.png">
<img width="1215" alt="Screenshot 2021-10-14 at 23 48 26" src="https://user-images.githubusercontent.com/30329690/137405927-3ce98546-a92f-4f8f-9647-a60582d7a895.png">

**Comment by sleepycorgi1** [comment, marked as answer]
Hey KrisWilka,
I believe I helped resolve this for you via discord. Please let me know if that's not the case.
Thank you.

**Comment by KrisWilka** [comment]
Yes they did thank you very much.
